src/EditorFeatures/Test/SolutionCrawler/WorkCoordinatorTests.cs

```diff
@@ -825,7 +825,7 @@ End Property
         private async Task InsertText(string code, string text, bool expectDocumentAnalysis, string language = LanguageNames.CSharp)
         {
```

Comment:
@CyrusNajmabadi @Pilchie wanted to make sure you saw the changes I made in this file. 

This test defined two exportable MEF components.  In the past they were loaded into the majority of composition due to a quirk of our loading code. A lot of our code in `TestExportProvider` specifies types which should load yet there are a lot of consumers which just convert all of those `Type` values into `Type.Assembly`.  Hence really when we say "type" should load we're saying "nah just load everything in this assembly".  

In the past this caused `WorkspaceWaiter` and `SolutionCrawlerWaiter` to be loaded into most compositions.  Once the assemblies split this stopped being true.  Had to create a custom `ExportProvider` for this test to get it working again.